src/event_pusher/mod_event_pusher_rabbit.erl

```diff
@@ -177,6 +182,17 @@ exchanges(Host) ->
      opt(Host, groupchat_msg_exchange, ?DEFAULT_GROUP_CHAT_MSG_EXCHANGE)
     ].
 
+-spec initialize_metrics(Host :: jid:server()) -> ok.
+initialize_metrics(Host) ->
+    [mongoose_metrics:ensure_metric(Host, Name, Type)
+     || {Name, Type} <- [{?RABBIT_CONNECTIONS_METRIC, spiral},
```

Comment:
Why not `gauge` for the number of connections?

Reply:
Cause `spiral` simply fits here. Using `spiral` metric I can just increase a counter when a worker established a connection and decrease when it closed one. Moreover, we use `spiral` for the same type of metric in Amoc.